README.md

```diff
@@ -1,47 +1,185 @@
+
+* Install functional tests Python requirements (mandatory before executing functional tests),
+* [Execute functional tests](#test),
+* Open a terminal in your docker container,
+* Clean build files,
+* Load the app on a physical device.
+
+You can find all the available tasks definitions in `.vscode/tasks.json`.
+
+### With a shell and the docker builder image
 
 The app-builder docker image [from this repository](https://github.com/LedgerHQ/ledger-app-builder) contains all needed tools and library to build and load an application.
 You can download it from the ghcr.io docker repository:
 
 ```shell
-sudo docker pull ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-full
+sudo docker pull ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder
 ```
 
 You can then enter this development environment by executing the following command from the directory of the application `git` repository:
 
+**Linux (Ubuntu)**
+
 ```shell
-sudo docker run --rm -ti --user "$(id -u)":"$(id -g)" -v "$(realpath .):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-full
+sudo docker run --rm -ti --user "$(id -u):$(id -g)" --privileged -v "/dev/bus/usb:/dev/bus/usb" -v "$(realpath .):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder
+```
+
+**macOS**
+
+```shell
+sudo docker run  --rm -ti --user "$(id -u):$(id -g)" --privileged -v "$(pwd -P):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder
+```
+
+**Windows (with PowerShell)**
+
+```shell
+docker run --rm -ti --privileged -v "$(Get-Location):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
 ```
 
 The application's code will be available from inside the docker container, you can proceed to the following compilation steps to build your app.
 
-### Without the docker image builder
+## Compilation and load
 
-Be sure to have your environment correctly set up (see [Getting Started](https://developers.ledger.com/docs/nano-app/introduction/)) and [ledgerblue](https://pypi.org/project/ledgerblue/) and installed.
+### Compilation
 
-If you want to benefit from [vscode](https://code.visualstudio.com/) integration, it's recommended to move the toolchain in `/opt` and set `BOLOS_ENV` environment variable as follows
+Once you have setup a working development environment (by following the [shell with docker](#with-a-shell-and-the-docker-builder-image) approach), you can build and load the app.
+
+From inside the container, use following line to build :
 
 ```shell
-BOLOS_ENV=/opt/bolos-devenv
+make DEBUG=1  # compile optionally with PRINTF
 ```
 
-and do the same with `BOLOS_SDK` environment variable
+You can choose which device to compile and load for by setting the `BOLOS_SDK` environment variable to the following values :
+
+* `BOLOS_SDK=$NANOS_SDK`
+* `BOLOS_SDK=$NANOX_SDK`
+* `BOLOS_SDK=$NANOSP_SDK`
+* `BOLOS_SDK=$STAX_SDK`
+
+By default this variable is set to build/load for Nano S.
+
+### Loading on a physical device
```

Comment:
same, we could first recommend to use the vscode load action here as you did for test section ?